### DISCUSSION TITLE
Glitching screen after update to Ubuntu 21.10

### DISCUSSION THREAD

**Discussion by Lu-ni** [i3/i3#4695] [Q&A]
I uploaded my computer to Ubuntu 21.10 and now i have a bug with I3. (I went back to gnome and everything work fine, It's only when I use I3 that's happening)Here's what's happening :
[recorded with my phone](https://imgur.com/a/i4UpdDx )
What's funny is when i try to record with a kazam (screencast) nothing looks wrong:
[kazam](https://imgur.com/a/mjSENPF)
I spent already a lot of time and can't figure whats wrong ... I not sure I should post that here but you're my last hope before I reinstall everything...

- The bug is show here on telegram, but happens on almost every application just not everywhere on the screen
     - chrome : when I put my mouse on favorites
     -  typora : all the text and the Table of content
     - terminal : Don't put the cursor at the correct place 

The log of I3wm show :
```
ERROR: Another window manager is already running (WM_Sn is owned)
But I can't find what other WM run..
```

I had i3 4.19 and uploaded it to 4.20. no changes
```
$  i3 --moreversion 2>&- || i3 --version
Binary i3 version:  4.20 (2021-10-19) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.20 (2021-10-19) (pid 2052)
Loaded i3 config:
  /home/bob/.config/i3/config (main) (last modified: Mo 11 Okt 2021 11:24:00, 3937573 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3
```

[my hardware](https://pastebin.com/DemNR1id)
[log of I3wm](https://pastebin.com/hLgVb8Jv)

Thank you for reading :)

**PARTIAL SOLUTION** : 
use _picom_ (compositor)
`exec --no-startup-id picom -bc`

**Comment by Airblader** [comment]
> Here's what's happening :

That looks like a graphics issue. If you're running a compositor, try disabling it (maybe vice versa). Make sure your graphics driver is up to date, too.

> The log of I3wm show :

That looks like you didn't run the correct command but rather just tried to start i3 again.

**Comment by Lu-ni** [comment]
I've been looking for a solution. Nothing came out. Another user has the same problem as me after updating ubuntu.
I don't have a compositor running and I'm starting I3 from the logging screen.

I really don't understand why it's doing that. Graphic card up to date and no bug when I'm not using I3.

**Comment by orestisfl** [comment]
Does this happen with other window managers? E.g. can you try with openbox?

**Reply by Lu-ni** [reply]
I tried with xmonad, same result. So, I think this doesn't really belong here. Where should I go to post this Issue ? (It's the update from 21.04 to 21.1 that broke it)

**Comment by CDSoft** [comment]
Hello. I had similar results when upgrading from Fedora 34 to 35. The problem was "fixed" by running a compositor (picom in my case). I just added `exec --no-startup-id picom -bc` to the config file.

**Reply by Lu-ni** [reply]
Thank you, it take care of 80% of the issue. Do you have any idea why it's doing that ?
